target/linux/realtek/files-6.12/drivers/net/pcs/pcs-rtl-otto.c

```diff
@@ -233,12 +233,12 @@ static int rtpcs_sds_write_bits(struct rtpcs_serdes *sds, int page,
 	return rtpcs_sds_write(sds, page, regnum, reg);
 }
 
-static int rtpcs_sds_modify(struct rtpcs_ctrl *ctrl, int sds, int page, int regnum,
+static int rtpcs_sds_modify(struct rtpcs_serdes *sds, int page, int regnum,
```

Comment:
Just a friendly reminder. sds_write_bits() and sds_modify() are basically the same functions. One uses a mask, the other high/low bit id. We should later decide which one to keep.